_posts/2025-01-12-AncientMetagenomeDir_hackathon.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "AncientMetagenomeDir - hackathon (Feb 2025)"
+date: 2025-02-07
+categories:
+  - Events
+  - News
+sidebar_link: true
+---
+
+<img src="/assets/media/spaam-AncientMetagenomeDir_socialmedia.png" alt="AncientMetagenomeDir logo" width="50%" class="center">
+
+The core team of AncientMetagenomeDir will organise another hackathon to tackle some of AncientMetagenomeDir's open issues and add new features to the database.
+
+Hackathons have helped to accelerate the progress of the AncientMetagenomeDir project in the past and have made it so successful. It is a fun way to work on a larger project with like-minded people collaboratively and to help a project take off. Instead of working all by yourself on an issue, you will have the opportunity to join others and work together in a (virtual) space. This year our main goal will be to kick-start the addition of C14 radiocarbon dates to the database.
+
+While we cannot award any monetary rewards, everyone who contributed will be part of future AncientMetagenomeDir publications, similarly to the two previously published, well-cited publications ([FellowsYates2021](https://doi.org/10.1038/s41597-021-00816-y) and [Borry2024](https://doi.org/10.12688/f1000research.134798.2)).
+
+## When?
+
+07th February 2025 - 9 am to 5 pm CET.
+
+In case there are volunteers from other time zones than Europe, we are happy to accommodate this. So please welcome to join from all across the world.
```

Comment:
In case there are volunteers from other time zones than Europe, we are happy to accommodate this.  We are please to welcome you from any part of the world and please reach out it you would like to host it in a different time zone.